aether-kernel/aether/kernel/api/avro_tools.py

```diff
@@ -469,7 +468,7 @@ def avro_schema_to_passthrough_artefacts(item_id, avro_schema):
         # this is an auto-generated mapping that shouldn't be modified manually
         'is_read_only': True,
         'is_active': True,
-        'input': sample,  # include a data sample
+        'input': definition,  # include a data sample
```

Comment:
I thought that the input was the data sample, not the AVRO schema itself... was I wrong?

Reply:
Was to be the AVRO schema